### PR TITLE
#156 update ico fix

### PR DESCRIPTION
**Pull request by jpmccu**

### CODE REVIEW

File: nginx/default.conf [hunk 1]
```diff
@@ -43,7 +43,11 @@ server {
   #   rewrite /whyis/(.*) /$1 break;
   #   proxy_pass http://whyis;
   # }
+<<<<<<< HEAD
+
+=======
  
+>>>>>>> develop
```
[the remark this answers is not in the document]
Comment (jpmccu) [reply nginx/default.conf]: Odd, that shouldn't have even shown up as a conflict. Grr.

**Review by stouffers — COMMENTED** [review]
Unsure why #156 is referenced by this PR.
It appears that this is completing integration of containerized whyis?

**Comment by tholulomo** [comment]
Okay, I will

**Comment by tholulomo** [comment]
Running into this 
Screen Shot 2022-07-21 at 1 55 09 PM

File: docker-compose.yml [hunk 2]
```diff
@@ -92,57 +92,62 @@ services:
...
+  celery:
+    depends_on:
+      - redis
+      - fuseki
+    volumes:
+      - ./mockDB/fuseki:/app/run
+      - ./whyis/materialsmine:/app
+    command: /opt/venv/bin/celery -A wsgi.celery worker -l INFO -c 4 --uid=nobody --gid=nogroup
+    environment:
+      - CHOKIDAR_USEPOLLING=true
```
[the remark this answers is not in the document]
Comment (jpmccu) [reply docker-compose.yml:110]: I think I left it in because it was in the original example. :-)

**Comment by tholulomo** [comment]
KG App image generated and now in docker hub. We will continue to keep this branch for future updates

**Pull request closed without merging**